spec/features/finance/npq/course_payment_breakdown_spec.rb

```diff
@@ -13,21 +13,27 @@
   let(:npq_course_leading_behaviour_culture)      { create(:npq_course, identifier: npq_leading_behaviour_culture_contract.course_identifier, name: "Leading Behaviour Culture") }
   let(:npq_course_leading_teaching_development)   { create(:npq_course, identifier: npq_leading_teaching_development_contract.course_identifier, name: "Leading Teaching Development") }
   let(:breakdowns) do
-    Finance::NPQ::CalculationOverviewOrchestrator.call(
+    Finance::NPQ::CalculationOverviewOrchestrator.new(
+      statement: statement,
+      aggregator: Finance::NPQ::ParticipantEligibleAndPayableAggregator,
+    ).call(event_type: :started)
+  end
+  let!(:statement) do
+    Finance::Statement::NPQ.create!(
+      name: "January 2022",
+      deadline_date: Date.new(2022, 1, 31),
+      payment_date: Date.new(2022, 2, 16),
       cpd_lead_provider: cpd_lead_provider,
-      event_type: :started,
     )
   end
-  let(:statement) { Finance::Statement::NPQ.find_by_name("payable") }
 
-  scenario "see a payment breakdown per NPQ course and a payment breakdown of each individual NPQ courses for each provider" do
+  xscenario "see a payment breakdown per NPQ course and a payment breakdown of each individual NPQ courses for each provider" do
```

Comment:
is the NPQ payment breakdown broken?

Reply:
I looked into this briefly when  adding to the ECF spec. I think it is likely an issue with the test setup

Reply:
Addressed in separate PR